k-distribution/src/main/scripts/lib/pyk/pyk/tests/test_pretty_print_kast.py

```diff
@@ -0,0 +1,21 @@
+from typing import Callable, Final, Mapping, Tuple
+from unittest import TestCase
+
+from pyk.kast import TRUE, KAst, KRule, prettyPrintKast
+
+
+class PrettyPrintKastTest(TestCase):
+    TEST_DATA: Final[Tuple[Tuple[KAst, str], ...]] = (
+        (KRule(TRUE), 'rule true'),
+        (KRule(TRUE, ensures=TRUE), 'rule true'),
+    )
+
+    SYMBOL_TABLE: Final[Mapping[str, Callable]] = {}
+
+    def test(self):
+        for i, (kast, expected) in enumerate(self.TEST_DATA):
+            with self.subTest(i=i):
+                actual = prettyPrintKast(kast, self.SYMBOL_TABLE)
+                actual_tokens = actual.split()
+                expected_tokens = expected.split()
+                self.assertListEqual(actual_tokens, expected_tokens)
```

Comment:
For now I wanted to ignore whitespace characters. But later on we can strengthen the tests to check them as well.

Reply:
`with self.subTest(i=i)` assigns an index to the given iteration so that if there is a failure the output will show which iteration the failure occurred in:

https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests

This looked like a good enough way to do parametric tests without introducing a new dependency.